modules/mockserver/src/test/java/org/testcontainers/containers/MockServerContainerTest.java

```diff
@@ -39,4 +39,12 @@ public void newVersionStartsWithDefaultWaitStrategy() {
             mockServer.start();
         }
     }
+
+    @Test
+    public void waitStrategyShouldWorkWithActiveTLS() {
+        try (MockServerContainer mockServer = new MockServerContainer(MOCKSERVER_IMAGE)) {
+            mockServer.withEnv("MOCKSERVER_TLS_MUTUAL_AUTHENTICATION_REQUIRED", "true");
+            mockServer.start();
```

Comment:
It will be great if a HTTP request is done and configured properly. Also, it can be added to the `mockserver.md` docs